backend/open_webui/utils/auth.py

```
@@ -8,7 +9,9 @@
 import os
 
 
-from datetime import UTC, datetime, timedelta
+from datetime import datetime, timedelta
+import pytz
```

Ruff (0.8.2)

`pytz` imported but unused

Remove unused import: `pytz`

(F401)